oceanprefilter/prefilter_test.go

```diff
@@ -0,0 +1,119 @@
+package oceanprefilter
+
+import (
+	"fmt"
+	"image"
+	"math"
+	"os"
+	"path/filepath"
+	"testing"
+	"image/jpeg"
+	xgb "github.com/Elvenson/xgboost-go"
+	"github.com/Elvenson/xgboost-go/activation"
+)
+
+
+func TestGeneralModel(t * testing.T) {
+
+	f, err := os.Open("numbered_data/1000.jpg")
+	fmt.Print(err)
+	defer f.Close()
+	img, _, err := image.Decode(f)
+
+	rc := runConfig{}
+	ensemble, err := xgb.LoadXGBoostFromJSONBytes(modelbytes,
+		"", 2, 8, &activation.Softmax{}) // wrong activation logistic is 1 class only
+	if err != nil {
+		panic(err)
+	}
+	rc.model = ensemble
+	rc.threshold = 0.25
+	rect := image.Rectangle{
+		Min: image.Point{X: 250, Y: 350},
+		Max: image.Point{X: 580, Y: 480},
+	}
+	rc.excludedZone = &rect
+	
+	res, err := make_inference(img, rc)
+	if err != nil {
+		fmt.Printf("err")
+	}
+
+	fmt.Printf("output result is %+v\n", res)
+}
+
+func TestAcc(t * testing.T) {
+	dir := "numbered_data/"
```

Comment:
You should upload the entire directory for the test, if anything, this test should become an example, and you can create an example program to run this. You can see how to organize the package to have an example program with this resource: https://www.digitalocean.com/community/tutorials/how-to-write-packages-in-go

Reply:
Goodness, I meant you SHOULDN'T upload the entire directory